5calls/app/src/main/java/org/a5calls/android/a5calls/model/DatabaseHelper.java

```diff
@@ -333,6 +364,37 @@ public List<Pair<String, Integer>> getCallCountsByIssue() {
         return result;
     }
 
+    public List<String> getStarredIssues() {
+        Cursor c = getReadableDatabase().rawQuery(
+                "SELECT " + StarredIssuesColumns.ID + " FROM " + STARRED_ISSUES_TABLE_NAME
+                + " ORDER BY " + StarredIssuesColumns.TIMESTAMP, null);
+        List<String> result = new ArrayList<>();
+        while (c.moveToNext()) {
+          result.add(c.getString(0));
+        }
+        c.close();
+        return result;
+    }
+
+    // Delete any starred issues not present in the provided list of issue ids to prevent the
+    // table from filling up with issues that don't exist any more so the app doesn't have to look
+    // through as many items when seeing if the issue needs to be marked as starred
+    public int trimStarredIssues(List<String> ids) {
```

Comment:
I don't know if we should do this -- i feel like there's a world in which we would bring back an old issue because something new came up. I can also imagine wanting to know what a user has starred in the future. 

If the concern is that the starred issues list is too long, can we instead say getStarredIssues(List<String> ids) and have the DB lookup just look through the starred ones?